content/concepts/transport.md

```diff
@@ -210,3 +210,76 @@ Following the multiaddress format described earlier, a standard QUIC connection
 look like: `/ip4/127.0.0.1/udp/65432/quic/`.
 
 In this section, we offered an overview of QUIC and how QUIC works in libp2p.
+
+## WebTransport
+
+Another transport protocol under development at the IETF is WebTransport.
+WebTransport is a new specification that uses QUIC to offer an alternative to
+WebSockets. It can be considered WebSockets over QUIC by allowing 
+browsers to establish a stream-multiplexed and bidirectional 
+connection to servers. 
+
+The specification can depend on and reuse the QUIC infrastructure in place 
+to offer WebSockets that take the benefits of UDP and offer streams without 
+head-of-line blocking.
+
+Recall that WebSockets are bidirectional, full-duplex communication between two 
+points over a single-socket connection. WebTransport can be used
+like WebSockets, but with the support of multiple streams.
+
+WebTransport supports reliable streams that can be arbitrary in size. They can be 
+independent and canceled if needed. The datagrams in a 
+WebTransport connections are MTU-sized.
+
+{{% notice "caution" %}}
+
+There is an experimental WebTransport transport in go-libp2p that is part 
+of the v0.23 release. The implementation should be used experimentally and is not 
+recommended for production environments.
```

Comment:
```suggestion
go-libp2p added (experimental) support for WebTransport transport as part
of the v0.23 release.
```

1. Can we also link to the release (notes)?
2. We should say that js-libp2p will release WebTransport support really soon, and maybe link to https://github.com/libp2p/js-libp2p-webtransport
3. We should say that there are currently no concrete plans to support WebTransport in any other libp2p implementation.